polar_route/mesh_generation/environment_mesh.py

```diff
@@ -204,9 +309,9 @@ def update_cellbox(self, index, values):
         if index > -1 or index < len(self.agg_cellboxes):
             self.agg_cellboxes[index].agg_data.update(values)
         else:
-            raise ValueError(f'Invalid cellbox index')
+            raise ValueError('Invalid cellbox index')
 
```

Comment:
Can the doc-string of this function be updated to include the new argument

Reply:
done